Tasks/Rusak/Task3/Task3-2.py

```diff
@@ -0,0 +1,36 @@
+text = str(input('Введите текст:\t'))
+text = text.lower()
+
+for char in text:
+    if not char.isalpha() and char != '-':
+        text = text.replace(char, ' ')
+
+list_text = text.split()
+
+for i in list_text:
+    if i=='':
+        list_text[i].remove('')
```

Comment:
could've add this check in the next cycle and do not spend time and memory on altering the data